database/factories/GiftQuizFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace EscolaLms\TopicTypeGift\Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use EscolaLms\TopicTypeGift\Models\GiftQuiz;
+
+class GiftQuizFactory extends Factory
+{
+    protected $model = GiftQuiz::class;
+
+    public function definition(): array
+    {
+        return [
+            'value' => $this->faker->text,
```

Comment:
better to add some real examples as this should pass validation and might break something https://docs.moodle.org/401/en/GIFT_format